src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Windows.cs

```diff
@@ -676,7 +690,7 @@ private static int GetNewBufferSize(int existingBufferSize, int requiredSize)
         private const int DefaultCachedBufferSize = 128 * 1024;
 #endif
 
-        private static unsafe ProcessInfo[] GetProcessInfos(IntPtr dataPtr)
+        private static unsafe ProcessInfo[] GetProcessInfos(IntPtr dataPtr, int processId = AllProcessIds)
```

Comment:
I think the only caller of this already defaults to AllProcessIds. So you could remove the default value.

Reply:
Thanks. Cleaned up now. I get confused by all those files and partial classes for all those different platforms to the extend that I don't quite understand anymore what's used from where and how... Dangerous setup, really.